snapshots/animal_welfare/2023-08-01/uk_egg_statistics.ods.dvc

```diff
@@ -1,17 +1,17 @@
 meta:
   origin:
-    dataset_title_owid: UK egg statistics
-    dataset_title_producer: UK egg statistics
-    dataset_description_producer: |
+    title_snapshot: UK egg statistics
```

Comment:
I suppose `title_snapshot` can be removed in this case, since it's identical to `title`.

Reply:
Fixed. (I haven't fixed YAML files as part of this PR. Wanted to leave it for the subsequent PR)